client/tailwind.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
       'ButtonText': 'ButtonText',
     },
     fontFamily: {
-      sans: 'var(--w-font-sans)',
+      sans: 'var(--w-font-sans,-apple-system,BlinkMacSystemFont,"Segoe UI",system-ui,Roboto,"Helvetica Neue",Arial,sans-serif,Apple Color Emoji,"Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji")',
```

Comment:
It might be nice to first work out why the fonts are not loading in the userbar CSS. 

After that - we may find we do not need this fallback. 

If we do however need it we should try to avoid hard-coding it and instead use the JS object/ array with them all defined.

Reply:
I think we can do this a different way, what if we instead set up a new utility?

```
    /**
     * SOME DESCRIPTION - this goes above the existing add variant
     */
    plugin(({ addUtilities }) => {
      addUtilities({
        'w-font-sans': { '--w-font-sans': fontFamily.sans.join(', ') },
      });
    }),
```

OR a custom component

```
    /**
     * SOME DESCRIPTION - this goes above the existing add variant
     */
    plugin(({ addComponents }) => {
      addComponents({
        '.reset-font-custom-properties': {
          '--w-font-sans': fontFamily.sans.join(', '),
        },
      });
    }),
```

This would then allow us to do the following in the userbar css.

```
.#{$namespace}-userbar-reset {
  all: initial;
  // Copy our font sans variable so it can be used without Tailwind
  @appy w-font-sans
..  OR
  @apply reset-font-custom-properties
```


Read a bit more here
* https://tailwindcss.com/docs/plugins#static-utilities
* https://tailwindcss.com/docs/functions-and-directives#tailwind

The goal being to avoid re-writing these font sets anywhere else except for the one place - declared in client/src/tokens/typography.js

Reply:
Oh, nice will look more on it

Reply:
This is also a nice idea, if we want to avoid redecorating the fonts.
I'll try to get a better approach now, than the hardcoding thing being done here.

Reply:
This won’t work because Tailwind isn’t loaded in the userbar stylesheet. I think there’s too much of a risk of it creating conflicts with people’s websites so I’d prefer we avoid it for the time being.